submissions/Roophee/Popup_checkbox/css/main.scss

```diff
@@ -0,0 +1,422 @@
+@import "_misc/normalize.scss";
+
+$text-color: #0c4e67;
+$hover-color: #01161e;
+
+:root {
+    font-size: 16px;
+    outline: none;
+}
+
+a.active.focus,
+a.active:focus,
+a.focus,
+a:active.focus,
+a:active:focus,
+a:focus,
+button.active.focus,
+button.active:focus,
+button.focus,
+button:active.focus,
+button:active:focus,
+button:focus,
+.btn.active.focus,
+.btn.active:focus,
+.btn.focus,
+.btn:active.focus,
+.btn:active:focus,
+.btn:focus {
+    outline: 0 !important;
+    outline-color: transparent !important;
+    outline-width: 0 !important;
+    outline-style: none !important;
+    box-shadow: 0 0 0 0 rgba(0, 123, 255, 0) !important;
+}
+
+*,
+*:before,
+*:after {
+    box-sizing: border-box;
+    -webkit-box-sizing: border-box;
+    -moz-box-sizing: border-box;
+}
+
+html,
+body {
+    width: 100%;
+    height: 100%;
+    padding: 0;
+    margin: 0;
+    font-family: Arial, Helvetica, sans-serif;
+    scroll-behavior: smooth;
+}
+
+.container {
```

Comment:
On fullHD screen I have that white space at the bottom of the page, Please fix it

![image](https://user-images.githubusercontent.com/16196199/101083125-30434600-35b5-11eb-9657-5bdbdc3862d6.png)

Reply:
Fixed. Set proper height of container. Thank you for your comment.